runner/src/main.rs

```diff
@@ -1098,7 +1103,9 @@ fn run_liche() -> Step {
     Step::Multiple {
```

Comment:
Isn't it more readable to just comment out the lines that call to `run_liche` and add the TODO there to uncomment them?

Reply:
Oh! Interesting. I can comment out the whole `run_liche` function, without getting any `clippy` complaints (I am used to style checks complaining about commented out **code**).